cookbooks/aws-parallelcluster-environment/kitchen.environment-config.yml

```diff
@@ -60,3 +60,37 @@ suites:
         directory_service:
           enabled: "true"
         node_type: HeadNode
+  - name: ephemeral_drives_mounted
+    driver:
+      instance_type: d3.xlarge  # instance type with ephemeral drives
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-environment::ephemeral_drives]
+    verifier:
+      controls:
+        - tag:config_ephemeral_drives_service_and_mount
+    attributes:
+      dependencies:
+        - recipe:aws-parallelcluster-environment::cfnconfig_mock
```

Comment:
Nit: we might want to use `mock` prefix instead of suffix to make it easier to search for mocks and to make the name more fluent in English (`mock_something`).

Reply:
addressed.